specs/sysext.md

```diff
@@ -43,6 +44,18 @@ Examples: `"SYSEXT_LEVEL=2"`, `"SYSEXT_LEVEL=15.14"`.
 
 If not present, and if `VERSION_ID=` is present instead, then this will be checked instead.
 
+#### `VERSION_ID=` `ID=`
+These are used for exact matches, and the meaning is defined in the
+[`os-release` specification](https://www.freedesktop.org/software/systemd/man/os-release.html).
```

Comment:
```suggestion
[`os-release` specification](https://www.freedesktop.org/software/systemd/man/os-release.html).
For the `ID=` field the special value `_any` can be set which matches any `os-release` value. The fields `VERSION_ID` or `SYSEXT_LEVEL` are not matched then.
```
Since the line above only talks about exact matches, I think it would be good to avoid conflicting statements/confusion with lines 31-34 by adding this clarification. Otherwise it sounds like `VERSION_ID` and `ID=` would always required to be exact matches.